plugins/action_change_status/src/changeStatus.ts

```diff
@@ -109,11 +109,23 @@ const changeReviewStatus = async (
     status: ActionQueueStatus.Fail,
     error_log: 'uknown error',
   }
-  console.log(`Changing the Status of Review ${reviewId}...`)
+  console.log(`Changing the Status of Review ${reviewId} to ${newStatus}...`)
 
   try {
     const currentStatus = await DBConnect.getReviewCurrentStatusHistory(reviewId)
 
+    if (newStatus === undefined) {
+      console.log(`WARNING: Review newStatus property is undefined. No changes were made.`)
+      returnObject.status = ActionQueueStatus.Success
+      returnObject.error_log = 'Status not changed'
+      returnObject.output = {
+        status: currentStatus.status,
+        statusId: currentStatus.status_history_id,
+        reviewStatusHistoryTimestamp: currentStatus.status_history_time_created,
+      }
+      return returnObject
+    }
+
```

Comment:
What was the reason for this? When does this action get run with `newStatus === undefined`?

Reply:
When you don't set the property `newStatus`, I wrongly set it as `status` and got it failing without understanding why!

Reply:
Okay, fair enough.

In that case, this should actually be an error:
```ts
      returnObject.status = ActionQueueStatus.Fail
      returnObject.error_log = 'Missing property: "newStatus"'
     // No returnObject.output
```

We don't want missing properties to silently not work, they should very clearly show an error and stop the action sequence.